### DISCUSSION TITLE
Submit to Strangeloop?

### DISCUSSION THREAD

**Discussion by jackiekazil** [Open-Source-Software-Neighborhood-Watch/town-square#9] [General]
I feel like we should submit to Strangeloop. Thoughts? 
https://thestrangeloop.com/cfp.html 

CFP open: March 2, 2022
CFP close: April 20, 2022
Speakers notified by: May 6, 2022
Preconference date: Sep 22, 2022
Conference dates: Sep 23-24, 2022

For one of these tracks: 
* Software with a Mission (SWAM) - using technology to support the mission of social causes, activist organizations, government institutions, and nonprofits
* Sustainable Open Source (SOS) - tools and approaches for sustainable open source (funding, tools, community support, etc)

**Comment by tpike3** [comment]
I agree that would be good

For SWAM... I could propose (although I think iqt and John have some better stuff)

- Democratic AI/Adaptable Apps as the idea of stored knowledge

For SOS.. I could propose...
- Understanding emergent organizations of open source structure (initial results)   (I think git Geo would be great here)

Of course we could just submit a wide slew of stuff and hope it all gets picked up.

**Comment by jspeed-meyers** [comment]
This is cool. From reading the CFP closely, it looks like talks should be by one person or two. One is preferred, according to the CFP.

If other people decide to propose a talk, I'll probably propose a "a dataset of software supply chain security compromises" talk based loosely on the IQT Labs software supply chain software supply chain compromise dataset and related papers.

**Comment by jspeed-meyers** [comment]
I submitted the "Huawei Question Comes to Open Source Software" topic.

**Comment by tpike3** [comment]
I submitted "Democracy Needs More Coding" topic
